src/layouts/partials/footer.html

```diff
@@ -97,6 +97,10 @@ <h1>DCRDEX</h1>
                             See Dev Docs → 
                         </a>
                     </p>
+
+                    <p>
+                        <a href="https://is.decred.online/status/dcrdex" target="_blank">DCRDEX Status</a>
```

Comment:
As this is an external link, please add `rel="noopener noreferrer"`

Reply:
Updated, thanks.